.cm/platform_labels.cm

```diff
@@ -110,9 +110,9 @@ platforms:
   - cc:
     name: 'configuration_cache'
     subprojects:
+      - 'platforms/core-configuration/configuration-cache/'
```

Comment:
❗ If the platform is going to be named `core-configuration`, then this should look like:

```
  - cc:
    name: 'core_configuration'
    subprojects:
      - 'platforms/core-configuration'
      - 'subprojects/api-metadata/'
...continues
```

There's no need to be more specific, everything under the `platforms/core-configuration` directory will match.

The `summary_table.cm` file should be adjusted to match this as well.

Reply:
Updated